README.md

```diff
@@ -272,7 +272,22 @@ it to be saved. *This has been fixed in PassportJS 0.3.0*
 This is the secret used to sign the session ID cookie. This can be either a string
 for a single secret, or an array of multiple secrets. If an array of secrets is
 provided, only the first element will be used to sign the session ID cookie, while
-all the elements will be considered when verifying the signature in requests.
+all the elements will be considered when verifying the signature in requests. The
+secret itself should be not easily parsed by a human and would best be a random set
+of characters. A best practice may include:
+
+  - The use of environment variables to store the secret, ensuring the secret itself
+    does not exist in your repository.
+  - Periodic updates of the secret, while ensuring the previous secret is in the
+    array.
+
+Using a secret that can be guessed will reduce the ability to hijack a session to
```

Comment:
will change from **can** to **cannot**

Reply:
No, can js what I mean. Changing it to cannot is wrong, as then it will say if you cannot guess the secret, then you can hijack the session with only guessing the session ID.

Reply:
Hey, it looks like this was never actually changed? I can't really view the entire file (`Commit range not found`), only the lines in the preview. 
The current phrasing led me to believe the signature doesn't prevent from guessing the session ID (if you consider the signature not part of the session ID), but from something else that isn't mentioned. which made me [asking on stackoverflow](https://stackoverflow.com/questions/64667592/what-is-the-purpose-of-the-secret-in-express-session-middleware).  
My solution would be the following phrasing, but I'm not a native english speaker (as you probably figured out):
>Using a secret that **cannot** be guessed will reduce the ability to hijack a session **by** only guessing the session ID (as determined by the genid option).